README.md

```diff
@@ -733,7 +733,56 @@ spec:
   containers:
   - name: main-container
     image: your-image:tag
- ```
+```
+
+You can also use common labels to set resource attributes. 
+
+The following labels are supported:
+- `app.kubernetes.io/name` becomes `service.name`
+- `app.kubernetes.io/version` becomes `service.version`
+- `app.kubernetes.io/part-of` becomes `service.namespace`
+- `app.kubernetes.io/instance` becomes `service.instance.id`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-pod
+  labels:
+    app.kubernetes.io/name: "my-service"
+    app.kubernetes.io/version: "1.0.0"
+    app.kubernetes.io/part-of: "shop"
+    app.kubernetes.io/instance: "my-service-123"
+spec:
+  containers:
+  - name: main-container
+    image: your-image:tag
+```
+
+This requires an explicit opt-in as follows:
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: my-instrumentation
+spec:
+  defaults:
+    useLabelsForResourceAttributes: true
+```
+
+#### Priority for setting resource attributes
+
+The priority for setting resource attributes is as follows (first found wins):
+
+- Resource attributes set via `OTEL_RESOURCE_ATTRIBUTES` and `OTEL_SERVICE_NAME` environment variables
+- Resource attributes calculated from the pod's metadata (e.g. `k8s.pod.name`)
+- Resource attributes set via the `Instrumentation` CR (in the `spec.resource.resourceAttributes` section)
+- Resource attributes set via annotations (with the `resource.opentelemetry.io/` prefix)
+- Resource attributes set via labels (e.g. `app.kubernetes.io/name`)
```

Comment:
let's change this to be a numbered list

Reply:
where does the new `useLabelsForResourceAttributes` fall into this? it is the last rule? if so, we should specify that you need to set defaults.useLabelsForResourceAttributes=true